test/ERC721Contract.ts

```diff
@@ -0,0 +1,48 @@
+import { loadFixture } from "@nomicfoundation/hardhat-network-helpers";
+import { expect } from "chai";
+import { ethers } from "hardhat";
+
+describe("ERC721Contract unit tests", function () {
+  const Name = "Name";
+  const Symbol = "Symbol";
+
+  async function deployMetazokuFixture() {
+    const [owner, address1, address2, address3] = await ethers.getSigners();
+
+    const ERC721Contract = await ethers.getContractFactory("ERC721Contract");
```

Comment:
Let's add contractFileName variable as we deed it in deploy script file. 

`const contractFileName = __filename.split("/").splice(-1)[0].split(".").splice(0)[0];`